index.js

```diff
@@ -52,4 +52,163 @@ const state = {
     }
   ],
   cart: []
-};
\ No newline at end of file
+};
+
+let fruitIsAlreadyInCart = false
+let index = 0
+
+function renderStoreIcons() {
+  const storeItemList = document.querySelector('.store--item-list');
+  storeItemList.innerHTML = ''
+  for (let item of state.items) {
```

Comment:
I have never seen this type of a "for" loop. it would be great to get some clarification on how it works.

Reply:
Just leaving an answer here after our chat, for history: state.items is an array, and "item" is just the current element of the array that the for loop loops through. "item" can be called anything, it's just there to be able to be referenced inside the for loop as "the current element you're on".